### ISSUE TITLE
how much is the loss?

### ISSUE THREAD

**Issue by 14211019** [brianhuang1019/FCN-pytorch#20]
I am training the net and i wangt to know the final BCEloss value  you guys got around,thanks!
